gap/PGMatrix.gd

```diff
@@ -0,0 +1,96 @@
+#!
+#!
+#! @Chapter Hyperbolic lattice symmetries
+
+#!
+#! @Section Point group matrices
+#! @SectionLabel PGMatrix
+#!
+#! Symmetry elements of triangle groups might act non-trivially on hyperbolic momenta. 
+#! As such, the degrees of freedom of observables and topological invariants, like the 
+#! first Chern numbers in momentum space, might be contstrained by the hyperbolic lattice 
+#! symmetries of underlying models. We can determine how hyperbolic momenta transform as 
+#! hyperbolic lattice symmetry transformations act on Abelian states. This can be extracted 
+#! by constructing point-group matrices.
```

Comment:
```suggestion
#! point-group matrices.
```

Reply:
It seems strange that above you are talking about generic symmetries and suddenly all of them have associated **point-group** matrices.